lib/init.coffee

```diff
@@ -10,10 +10,13 @@ checkedAppend = (parameters, opt, args) ->
     parameters.push opt
     parameters.push(args...)
 
-makeParameters = (globals, ignore, file) ->
+makeParameters = (globals, ignore, std, file) ->
   parameters = ['-', '--no-color', '--codes', '--ranges', "--filename=" + file]
   checkedAppend parameters, '--globals', globals
   checkedAppend parameters, '--ignore', ignore
+  if std.length > 0
+    parameters.push '--std'
+    parameters.push(std.join '+')
```

Comment:
Ah, I like that way even better as we aren't _forcing_ people to expand on the default, letting them completely override it if so desired.